packages/nextjs/components/ExampleUi/ContractInteraction/CopyIcon.tsx

```diff
@@ -0,0 +1,28 @@
+type CopyIconProps = {
+  className: string;
+};
```

Comment:
nitpick: Maybe we inline this directly? `({ className }: { className: string }) `.

having the type seems a bit verbose